### ISSUE TITLE
Error notice caught by WP fatal error protection

### ISSUE THREAD

**Issue by danielbachhuber** [cedaro/satispress#98]
WordPress sent me an email with this notice and stacktrace:

```
An error of type E_ERROR was caused in line 189 of the file /nas/content/live/site/wp-content/plugins/satispress/src/Provider/Authentication.php. Error message: Uncaught TypeError: strpos() expects parameter 1 to be string, null given in /nas/content/live/site/wp-content/plugins/satispress/src/Provider/Authentication.php:189
Stack trace:
#0 /nas/content/live/site/wp-content/plugins/satispress/src/Provider/Authentication.php(189): strpos(NULL, '/')
#1 /nas/content/live/site/wp-content/plugins/satispress/src/Provider/Authentication.php(164): SatisPress\Provider\Authentication->get_request_path()
#2 /nas/content/live/site/wp-content/plugins/satispress/src/Provider/Authentication.php(75): SatisPress\Provider\Authentication->is_satispress_request()
#3 /nas/content/live/site/wp-content/plugins/satispress/vendor/cedaro/wp-plugin/src/AbstractPlugin.php(180): SatisPress\Provider\Authentication->register_hooks()
#4 /nas/content/live/site/wp-content/plugins/satispress/satispress.php(74): Cedaro\WP\Plugin\AbstractPlugin->register_hooks(Object(SatisPress\Provider\Authentication))
#5 /nas/content/live/site/wp-settings.
```

**Comment by bradyvercher** [comment]
@danielbachhuber Thanks for the report and PR. Is this a frequent notice or just occasional?

**Comment by danielbachhuber** [comment]
@bradyvercher I only received the email once, but it's the new error reporting feature that was released in WordPress 5.2. So, I don't know whether it's a frequent notice or not.
